jbehave-support-core/src/main/java/org/jbehavesupport/core/JBehaveDefaultConfig.java

```diff
@@ -142,7 +143,12 @@ public WebDriver webDriver(WebDriverFactoryResolver webDriverFactoryResolver) {
         return (WebDriver) proxyFactory.getProxy();
     }
 
+    @Bean
+    public FailScreenshotsReporterExtension failScreenshotsReporterExtension() { return new FailScreenshotsReporterExtension(); }
```

Comment:
Do we want these extensions exposed to user as default? (right now I think we dont register any reporting extensions and leave that to user)

Reply:
Sorry this is still WIP thing.